pkg/gui/controllers/remote_branches_controller.go

```diff
@@ -187,3 +197,14 @@ func (self *RemoteBranchesController) newLocalBranch(selectedBranch *models.Remo
 func (self *RemoteBranchesController) checkoutBranch(selectedBranch *models.RemoteBranch) error {
 	return self.c.Helpers().Refs.CheckoutRemoteBranch(selectedBranch.FullName(), selectedBranch.Name)
 }
+
+func (self *RemoteBranchesController) notMergingIntoYourself(branch *models.RemoteBranch) *types.DisabledReason {
```

Comment:
This check is not necessary: it's impossible to merge into yourself if you're merging a remote branch